Changing the process name breaks applying libcgroup rules, so don't do it.

diff -ru -x '*~' Starman-0.2014-orig/lib/Starman/Server.pm Starman-0.2014/lib/Starman/Server.pm
--- Starman-0.2014-orig/lib/Starman/Server.pm	2011-08-27 01:19:43.000000000 +0200
+++ Starman-0.2014/lib/Starman/Server.pm	2011-11-25 13:30:24.570434653 +0100
@@ -100,7 +100,7 @@
 
 sub run_parent {
     my $self = shift;
-    $0 = "starman master " . join(" ", @{$self->{options}{argv} || []});
+    #$0 = "starman master " . join(" ", @{$self->{options}{argv} || []});
     $self->SUPER::run_parent(@_);
 }
 
@@ -113,7 +113,7 @@
         DEBUG && warn "[$$] Initializing the PSGI app\n";
         $self->{app} = $self->{options}->{psgi_app_builder}->();
     }
-    $0 = "starman worker " . join(" ", @{$self->{options}{argv} || []});
+    #$0 = "starman worker " . join(" ", @{$self->{options}{argv} || []});
 }
 
 sub post_accept_hook {
